### ISSUE TITLE
SIC rolling video(s)

### ISSUE THREAD

**Issue by JimCircadian** [icenet-ai/aiuk-2023#2]
Alongside #1 it will be good if we can have SIC videos available (produced easily via `icenet_video_data` from OSISAF, but they'll be naff resolution) potentially alongside other variables that show the NH/SH cyclical SIC.

To clear these up it might be worth interpolating and adding a nicer land mask for presentations sake.

CC @MartinSJRogers

**Comment by MartinSJRogers** [comment]
These have now been produced for the Northern hemisphere using the polygonised landmasks built into the cartopy python package. 
Example below and code to make videos can be found at this personal repo: https://github.com/MartinSJRogers/osisaf_time_series 


https://user-images.githubusercontent.com/43956226/226562590-6481bb16-2796-4cd0-9906-a6b58b7ac7b1.mp4

**Comment by MartinSJRogers** [comment]
Happy for this issue to be closed now if you are, @JimCircadian

**Comment by JimCircadian** [comment]
Absolutely, lovely work @MartinSJRogers: the videos are entrancing people at the stand 😉

**Comment by MartinSJRogers** [comment]
Great, if you need me to redo with psychedelic colours to entrance them any more, let me know :)
